codegen/generator/nodejs/templates/src/method_comment.ts.tmpl

```diff
@@ -1,14 +1,21 @@
 {{ define "method_comment" }}
-	{{- if . -}}
-		{{- $commentLines := CommentToLines . }}
-		{{- /* we split the comment string into a string slice of one line per element */ -}}
+  {{- if .Description -}}
```

Comment:
@dolanor Please take a look! Is this the correct pattern?

Reply:
It would be.

Question is, what if there is no description but it is deprecated? Then we would generate nothing.

I guess we would need something like that:
```suggestion
  {{- if or .Description .DeprecationReason -}}
```